app/src/main/java/org/dhis2/data/filter/GetFiltersApplyingWebAppConfig.kt

```diff
@@ -0,0 +1,17 @@
+package org.dhis2.data.filter
+
+import org.dhis2.utils.filters.FilterItem
+import org.hisp.dhis.android.core.settings.FilterSetting
+
+class GetFiltersApplyingWebAppConfig {
+
+    fun <T> execute(
```

Comment:
If it is only used in FilterRepository and it's test, why is not a private method of it?